cloud/blockstore/libs/storage/partition_nonrepl/part_nonrepl_rdma_ut.cpp

```diff
@@ -571,6 +574,18 @@ Y_UNIT_TEST_SUITE(TNonreplicatedPartitionRdmaTest)
 
         UNIT_ASSERT_VALUES_EQUAL(2048 * 4096, counters.BytesWritten.Value);
         UNIT_ASSERT_VALUES_EQUAL(512 * 4096, counters.BytesRead.Value);
+
+        auto& requestCounters =
+            env.StorageStatsServiceState->Counters.RequestCounters;
+        UNIT_ASSERT_VALUES_EQUAL(
+            512 * 4096,
+            requestCounters.ReadBlocks.GetRequestBytes());
+        UNIT_ASSERT_VALUES_EQUAL(
+            0,
+            requestCounters.ReadBlocks.GetRequestVoidBytes());
+        UNIT_ASSERT_VALUES_EQUAL(
+            0,
+            requestCounters.ReadBlocks.GetRequestNonVoidBytes());
```

Comment:
let's move this triple assert to function which we can use everywhere and document that for rdma the counters are zero because the zero data is actually transferred on the line

Reply:
Не актуально, теперь там разные циферки, так как сделал статистику